frontend/docsettings.lua

```diff
@@ -188,6 +240,10 @@ function DocSettings:open(doc_path)
         dir_sidecar_file or "",
         -- Backup file of new sidecar file in docsettings folder
         dir_sidecar_file and (dir_sidecar_file..".old") or "",
+        -- Hash or PDF fingerprint-based sidecar file lookup
+        new.hash_sidecar_file or "",
+        -- Backup file of hash or PDF fingerprint-based sidecar file lookup
+        new.hash_sidecar_file and (new.hash_sidecar_file..".old") or "",
```

Comment:
Any reason we see here `new.` in `new.hash_sidecar_file` while the above don't have a `new.`, but only `dir_sidecar_file` ?

Reply:
For example:
```
    if lfs.attributes(new.dir_sidecar_dir, "mode") == "directory" then
        dir_sidecar_file = new.dir_sidecar_file
    en
```
I don't think it really matters that these have `new.` and the others don't?

About this candidate list functionality—if somehow a PDF gets in a scenario where it has multiple copies of metadata, e.g. one in `doc` and one in `hash`, then will it overwrite `hash` with `doc` (since `doc` was opened)? Is there or should there be any indication of overwriting or of multiple copies? Or is this an extreme enough edge case that it doesn't matter?

Reply:
> I don't think it really matters that these have new. and the others don't?

I don't know :) Not familier with all that is happening in there - but then, seeing some stuff with `new.` and some stuff without, I would be wondering if this difference means something :)

> About this candidate list functionality—if somehow a PDF gets in a scenario where it has multiple copies of metadata, e.g. one in doc and one in hash, then will it overwrite hash with doc (since doc was opened)? Is there or should there be any indication of overwriting or of multiple copies? Or is this an extreme enough edge case that it doesn't matter?

@hius07 knows more about that.
But I think we check the existence of all possible candidate, sort them by most recent, use the most recent - and when saving, save to preferred location and delete all other candidates found.

Reply:
> we check the existence of all possible candidate, sort them by most recent, use the most recent - and when saving, save to preferred location and delete all other candidates found

Exactly.